katalog/tests/harbor/config/Furyfile.yml

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020 SIGHUP s.r.l All rights reserved.
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
```

Comment:
Ingress 1.8.x was around the Kubernetes 1.19 time. Maybe we can bump this to a newer version